sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -272,8 +272,14 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
       }
       DescribeTableExec(desc.output, r.table, isExtended) :: Nil
 
-    case DescribeColumn(_: ResolvedTable, _, _) =>
-      throw new AnalysisException("Describing columns is not supported for v2 tables.")
+    case desc @ DescribeColumn(_: ResolvedTable, column, isExtended) =>
+      column match {
```

Comment:
Is it possible to have `GetArrayStructFields` here?

Reply:
Thanks for the catch. In this case, I will match against non-`Attribute` for nested types similar to v1:
https://github.com/apache/spark/blob/10b6466e91d2e954386c74bf6ab7d94f23dd6810/sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala#L768-L772